bundle/libraries/match_test.go

```diff
@@ -74,9 +74,9 @@ func TestValidateEnvironmentsNoFile(t *testing.T) {
 		},
 	}
 
-	diags := bundle.Apply(context.Background(), b, ValidateLocalLibrariesExist())
+	diags := bundle.Apply(context.Background(), b, ExpandGlobReferences())
 	require.Len(t, diags, 1)
-	require.Equal(t, "file ./wheel.whl is referenced in environments section but doesn't exist on the local file system", diags[0].Summary)
+	require.Equal(t, "no matching files for ./wheel.whl", diags[0].Summary)
```

Comment:
The new error message is a bit less clear, IMO.

Could we perhaps detect if a string is a glob pattern or not and use that to return different messages? E.g. if it's a pattern, we can say "no files match pattern", and if it isn't we can say "file doesn't exist".